browser-extension/src/components/ProfilePage.tsx

```diff
@@ -19,30 +18,19 @@ const ProfilePage: FC = () => {
       setProfile(producer);
       setUpdateProfile(producer);
     } catch (error) {
-      if (axios.isAxiosError(error)) {
-        const axiosError = error as AxiosError;
-        alert(`${axiosError.status}:  ${axiosError.message}`);
-      } else {
-        alert(`Error ${error}`);
-      }
+      client.displayError(error);
     }
   };
 
   const updateProducer = async () => {
     try {
       if (updateProfile) {
-        await client.updateProducer(updateProfile);
-        // const producer: Producer = await client.getProducer(); // put back later
-        setProfile(updateProfile);
-        console.log("saved");
+        const updatedProfile = await client.updateProducer(updateProfile);
+        setProfile(updatedProfile);
+        alert("Saved");
```

Comment:
not relevant to this PR but should we make it a task to make a prettier "alert" for our UI